tests/test_dm_control_env.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from garage.baselines import LinearFeatureBaseline
+from garage.envs.dm_control import DmControlEnv
+from garage.tf.algos import TRPO
+from garage.tf.envs import TfEnv
+from garage.tf.policies import GaussianMLPPolicy
+
+
+class TestDmControlEnv(unittest.TestCase):
+    def test_dm_control_env(self):
+        env = TfEnv(DmControlEnv(domain_name='cartpole', task_name='balance'))
```

Comment:
This can be one test case (algo integration test)

Can you also add parameterized tests (which cover every dm_control env) like the ones I wrote for gym?
* Test pickling/unpickling
* Test each env can be wrapped, constructed, stepped, rendered, and closed.
* Test the types observation_space, action_space, and observations produced by step/reset for the wrapped version.